docs/source/compression/compression_config_list.rst

```diff
@@ -1,11 +1,190 @@
 Compression Config Specification
 ================================
 
+Common Keys in Config
+---------------------
+
+op_types
+^^^^^^^^
+
+A list of type names of classes that inherit from ``torch.nn.Module``.
+Only module types in this list can be selected to be compressed.
+If this key is not set, all module types can be selected.
+
+op_names
+^^^^^^^^
+
+A list of fully-qualified name of modules that will be compressed.
+If the name referenced module is not existed in the model, it will be ignored.
+
+op_names_re
+^^^^^^^^^^^
+
+A list of regular expressions for matching module names.
+The matched modules will be selected to be compressed.
+
+exclude_op_types
+^^^^^^^^^^^^^^^^
+
+A list of type names of classes that inherit from ``torch.nn.Module``.
+Module types in this list can not be selected to be compressed.
+
+exclude_op_names
+^^^^^^^^^^^^^^^^
+
+A list of fully-qualified name of modules that will be excluded.
+
+exclude_op_names_re
+^^^^^^^^^^^^^^^^^^^
+
+A list of regular expressions for matching module names.
+The matched modules will be removed from the modules that need to be compressed.
+
+target_names
+^^^^^^^^^^^^
+
+A list of legal compressed target name, i.e., usually ``_input_``, ``weight``, ``bias``, ``_output_`` are support to be compressed.
+
+Two kinds of target are supported by design, module inputs/outputs(should be a tensor), module parameters/buffers:
+
+- Inputs/Outputs: If the module inputs or outputs is a singal tensor, directly set ``_input_`` for input and ``_output_`` for output.
+  ``_input_{position_index}`` or ``_input_{arg_name}`` can be used to specify the input target,
+  i.e., for a forward function ``def forward(self, x: Tensor, y: Tensor, z: Any): ...``, ``_input_0`` or ``_input_x`` can be used to specify ``x`` to be compressed,
+  note that ``self`` will be ignored when count the position index.
+  Similarly, ``_output_{position_index}`` can be used to specify the output target if the output is a ``list/tuple``,
+  ``_output_{dict_key}`` can be used to specify the output target if the output is a ``dict``.
```

Comment:
it is a little confusing here, because function output is always tuple, it is a list of variables. for `_output_{dict_key}`, it is a variable, you support open this dict variable to access its key/value. what if I return (list, dict) two variables, do you still open that dict?

Reply:
no, only single `tuple` and single `dict` (tuple with only one `dict` as item?) as output are supported.

Reply:
ok